oci/private/image.bzl

```diff
@@ -175,4 +178,5 @@ oci_image = rule(
         "@rules_oci//oci:registry_toolchain_type",
         "@aspect_bazel_lib//lib:jq_toolchain_type",
     ],
+    provides = [OciImageInfo],
```

Comment:
I wonder if `oci_image_index` should also provide this?

Reply:
I feel like we can do this without providers. simply looking into the extension?

Reply:
if the file ends with `.tar` then we assume it's an oci_tarball?

Reply:
okay